core/templates/dev/head/dashboard/notifications_dashboard.html

```diff
@@ -26,46 +26,47 @@ <h2 class="oppia-page-heading">Notifications</h2>
       Last updated: <[getLocaleAbbreviatedDatetimeString(jobQueuedMsec)]>
     </div>
 
-    <md-card class="oppia-page-card oppia-notifications-dashboard-card">
-      <div ng-if="recentNotifications.length === 0">
+    <div class="oppia-notifications-tray">
+      <md-card ng-if="recentNotifications.length === 0" class="oppia-page-card oppia-notifications-dashboard-card">
         <em>No recent notifications.</em>
-      </div>
+      </md-card>
 
       <div ng-if="recentNotifications.length > 0">
-        <table class="table">
-          <tr>
-            <th></th>
-            <th></th>
-            <th>Exploration</th>
-            <th>Last updated</th>
-            <th>by</th>
-            <th>Subject</th>
-          </tr>
-          <tr ng-repeat="notification in recentNotifications"
-              ng-class="{'oppia-dashboard-row-recent': lastSeenMsec && lastSeenMsec < notification.last_updated_ms && notification.author_username !== currentUsername, 'oppia-dashboard-row': !lastSeenMsec || lastSeenMsec >= notification.last_updated_ms || notification.author_username === currentUsername}"
-              ng-click="navigateToItem(notification.activity_id, notification.type)">
-            <td>
-              <span class="label label-primary"
-                    ng-if="lastSeenMsec && lastSeenMsec < notification.last_updated_ms && notification.author_username !== currentUsername">
-                New
-              </span>
-            </td>
-            <td>
-              <span ng-if="notification.type === 'exploration_commit'"
-                    class="glyphicon glyphicon-flag"
-                    title="Exploration commit"></span>
+        <md-card ng-repeat="notification in recentNotifications"
```

Comment:
If you give the md-card a style of "overflow: auto;", does that fix the floating problem?